app/components/mandatenbeheer/bestuursorgaan-mandaten.hbs

```diff
@@ -0,0 +1,100 @@
+<div class="au-o-box no-pagination" {{did-insert this.initialize}}>
+  <AuDataTable
+    @content={{this.orderedMandaten}}
+    @isLoading={{this.loading}}
+    @noDataMessage="Geen mandaten gevonden"
+    @page={{0}}
+    @size={{100}}
+    as |t|
+  >
+    <t.content as |c|>
+      <c.header>
+        <th>Naam</th>
```

Comment:
It would have meant adding query params and stuff to the route and i felt this would take me too far from the goal of this story. Decided not to implement it yet, especially since the routing may be revised later